zpages/zpages.go

```diff
@@ -31,16 +31,31 @@
 package zpages // import "go.opencensus.io/zpages"
 
 import (
+	"fmt"
 	"net/http"
+	"sync"
+
+	"go.opencensus.io/internal"
 )
 
-// Handler is an http.Handler that serves the zpages.
+// Handler is deprecated: Use NewHandler.
 var Handler http.Handler
```

Comment:
If this were changed to http.ServerMux I think that would work for what I referenced in #783. The http.Handler strips the `.Handle*` functions and while I could simply cast this to a ServerMux it feels dirty as I'm relying on knowledge of the implementation detail rather than a public contract.